### PR TITLE
Feature/lottery view

### PR DESCRIPTION
**Pull request by whereiswolf**
image

Done:
- Lottery view
- Settings widget
- Social Panel
- Description widget
- Copying link

To do (will be done in next PR):
- Sharing on Fb and Twitter
- Displaying list of elements
- Displaying results

### CODE REVIEW

File: src/components/CardWidget/CardWidget.style.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+  position: relative;
+  transition: all 0.3s;
+  width: 100%;
+  overflow-wrap: break-word;
+`;
+
+export const TitleWrapper = styled.div`
+  display: flex;
+  justify-content: row;
+  align-itmes: center;
```
Comment (lukix) [inline src/components/CardWidget/CardWidget.style.js]: `align-itmes` -> `align-items`

File: src/config/translations.js [hunk 2]
```diff
@@ -35,7 +42,7 @@ export default {
       INVALID_RANGE: 'Max value must be greater that the min value.',
     },
     LotterySettings: {
-      ELEMENTS_NUMBER: 'Number of elements to draw:',
+      ELEMENTS_NUMBER: 'Amount of elements to draw',
```
Comment (lukix) [inline src/config/translations.js]: From what I know, "amount" should only be used with uncountable nouns

File: src/config/translations.js [hunk 3]
```diff
@@ -80,11 +133,47 @@ export default {
       NEW_ITEM: 'Nowy Element',
       ELEMENTS_SET_A: 'Zbiór A',
       ELEMENTS_SET_B: 'Zbiór B',
-      NUMBER_OF_INTEGERS: 'Liczba losowanych elementów',
+      NUMBER_OF_INTEGERS: 'Ilość losowanych liczb',
```
Comment (lukix) [inline src/config/translations.js]: Same in Polish (liczba/ilość). However, I understand that "Liczba losowanych liczb" doesn't sound good.

File: src/redux/reducers/lottery.js [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+const initialState = {
+  isFetchingLottery: false,
+  isSavingNewLottery: false,
+  lottery: {},
+  error: {},
+};
+
+export default (state = initialState, action) => {
+  switch (action.type) {
+    case SAVE_NEW_LOTTERY_REQUEST: return Object.assign({}, state, {
```
Comment (lukix) [inline src/redux/reducers/lottery.js]: I prefer using object spread operator (`...`), but... whatever.

File: src/components/CardWidget/CardWidget.style.js (hunk 1 above)
Comment (whereiswolf) [reply src/components/CardWidget/CardWidget.style.js]: Czyli zbędne xd

File: src/redux/reducers/lottery.js (hunk 4 above)
Comment (whereiswolf) [reply src/redux/reducers/lottery.js]: ciiii 🤫

**Pull request merged**